packages/components/src/form-file-upload/test/index.js

```diff
@@ -14,15 +14,9 @@ import FormFileUpload from '../';
  */
 const { File } = window;
 
-function render( jsx ) {
-	return {
-		user: userEvent.setup( {
-			// Avoids timeout errors (https://github.com/testing-library/user-event/issues/565#issuecomment-1064579531).
-			delay: null,
-		} ),
-		...RTLrender( jsx ),
-	};
-}
+const user = userEvent.setup( {
```

Comment:
`@testing-library` recommends that `setup()` is called inline once per test.

https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent

Reply:
I must have missed that part when looking at the new `v14` docs, thank you for flagging it! Opened #40839 as a follow-up with those changes